pom.xml

```diff
@@ -43,8 +43,8 @@
 		<hsac-fitnesse-plugin.version>1.31.0</hsac-fitnesse-plugin.version>
 		<fitnesse.version>20211030</fitnesse.version>
 		<praegus-toolchain-fitnesse-plugin.version>2.0.14</praegus-toolchain-fitnesse-plugin.version>
-		<selenium.version>3.141.59</selenium.version>
-		<slf4j.version>1.7.32</slf4j.version>
+		<selenium.version>4.1.2</selenium.version>
+		<slf4j.version>1.7.25</slf4j.version>
```

Comment:
Any particular reason to downgrade slf4j?

Reply:
was meant to be .35 but I think i typoed. Will update